gym/error.py

```diff
@@ -168,83 +123,69 @@ def __init__(
         json_body=None,
         headers=None,
     ):
+        """Initialises the invalid request error."""
         super().__init__(message, http_body, http_status, json_body, headers)
         self.param = param
 
 
 class AuthenticationError(APIError):
-    pass
+    """Unused error."""
 
 
 class RateLimitError(APIError):
-    pass
+    """Unused error."""
 
 
 # Video errors
 
 
 class VideoRecorderError(Error):
-    pass
+    """Unused error."""
 
 
 class InvalidFrame(Error):
-    pass
+    """Error message when an invalid frame is captured."""
 
 
 # Wrapper errors
 
 
 class DoubleWrapperError(Error):
-    pass
+    """Unused error."""
 
 
 class WrapAfterConfigureError(Error):
-    pass
+    """Unused error."""
 
 
 class RetriesExceededError(Error):
-    pass
+    """unused error."""
```

Comment:
Capitalization

Reply:
For all of the unused error, I was tempted to delete them all but just wanted to confirm their won't be an issue with this.

Reply:
https://github.com/openai/gym/blame/master/gym/error.py
The error looks like they were added in the project's initial commits, wow

Im going to add a deprecation warning to them for gym 1.0 I think
This is the simplest way of removing without causing problems to other projects

Reply:
Done in eaf3a4c6e2dde0a128761ed50b4d9b884f575d8b